### DISCUSSION TITLE
Adaptation for Sofar HYD xxKTL-3PH

### DISCUSSION THREAD

**Discussion by rspafke** [cmcgerty/Sofar2mqtt#18] [Ideas]
I have received from Sofar the documentation for the Sofar HYD  5-20KTL-3PH modbus.  If someone wants to adapt the program for this inverter series I will gladly provide the documentation. I am not a programmer and therefore can not do it myself.

**Comment by cliffdude** [comment]
I have seen a few people asking for this inverter. If you could post a link then the devs could check it out and see what would be necessary to implement support for this model.

**Reply by wichr** [reply]
This version suport Sofar HYD KTL. https://github.com/IgorYbema/Sofar2mqtt
